crates/plex-api/tests/server.rs

```diff
@@ -925,6 +931,207 @@ mod online {
         assert!(matches!(err, Error::ItemNotFound));
     }
 
+    #[plex_api_test_helper::online_test]
+    async fn download(#[future] server: Server) {
+        // The first 32 bytes of white_noise_720p.mkv
+        const MKV_START: [u8; 32] = [
```

Comment:
What do you think about using `include_bytes!()` instead of hardcoding the bytes? The resulting binary for the test will be 20-something megabytes, but do we care? :)

Reply:
Oh I had not come across this macro, yeah that makes more sense.